tools/latest-kernel-full-config.sh

```diff
@@ -76,89 +76,18 @@ merge_kernel_configs() {
-    fi
 
     tmpdir=$(mktemp -d)
-    cp "${rpm_file}" "${tmpdir}/"
-    pushd "${tmpdir}" || bail "Could not move around"
-
-    rpm2cpio "${rpm_file}" | cpio -iu {,./}linux-"${version}".tar{,.xz} {,./}config-x86_64 {,./}config-aarch64 {,./}"*.patch" {,./}"${spec_file}"
-    # Upstream source is either xz compressed tarball or plain tarball
-    if [ -f "./linux-${version}.tar" ]; then
-        tar -xof linux-"${version}".tar; rm linux-"${version}".tar
-    else
-        tar -xof linux-"${version}".tar.xz; rm linux-"${version}".tar.xz
-    fi
-
-    # Find patch ordering based on the upstream SRPM and apply in that order
-    readarray -t patches < <(grep -P "^Patch\d+" "${spec_file}" | sort -n -k1.6 | grep -oP "^Patch\d+: \K.*\.patch$" "${spec_file}")
-
-    pushd "linux-${version}" || bail "Could not move into linux-${version}"
-
-    # Patches from the upstream
-    for patch in "${patches[@]}"; do
-        patch -p1 <"../$patch"
-    done
-
-    # Patches from bottlerocket
-    for patch in "${br_patches[@]}"; do
-        echo "Applying ${patch}"
-        patch -p1 <"$patch"
-    done
-
-    popd || bail "Could not move around - 'popd' failed. Lets stop before we break anything further."
-
-    for arch in "x86_64" "aarch64"; do
-
-        br_cfg="${kernel_path}/config-bottlerocket"
-        microcode_cfg="${kernel_path}/../microcode/${microcode_file}"
-        al_cfg="${PWD}/config-${arch}"
-        linux_src="${PWD}/linux-${version}"
-
-        pushd "linux-${version}" || bail "Could not move into linux-${version}"
-
-        if [ "${arch}" = "aarch64" ]; then
-            karch="arm64"
-            script_args=("../config-${arch}" "../config-bottlerocket")
-        elif [ "${arch}" = "x86_64" ]; then
-            karch="x86"
-            script_args=("../config-${arch}" "../config-microcode" "../config-bottlerocket")
-        fi
-
-        # mount config files and start the sdk docker container
-        docker run --rm \
-            -v "${br_cfg}":/config-bottlerocket \
-            -v "${microcode_cfg}":/config-microcode \
-            -v "${al_cfg}":/config-${arch} \
-            -v "${linux_src}":/linux-"${version}" \
-            -e ARCH="${karch}" \
-            -e CROSS_COMPILE=/usr/bin/${arch}-bottlerocket-linux-gnu- \
-            -e KCONFIG_CONFIG=bottlerocket_${arch}_defconfig \
-            -w /linux-"${version}" \
-            -u "$(id -u)":1000 \
-            --name "${arch}-kernel-${version}-config-merger" \
-            "${sdk_image}" \
-            ./scripts/kconfig/merge_config.sh \
-            "${script_args[@]}"
-
-        mv -f "bottlerocket_${arch}_defconfig" "${kernel_path}/config-full-bottlerocket-${arch}"
-        popd || bail "Could not move around - 'popd' failed. Lets stop before we break anything further."
-    done
-
-    popd || bail "Could not move around - 'popd' failed. Lets stop before we break anything further."
-
+    cp "${rpm_file}" "${tmpdir}/kernel-source.rpm"
```

Comment:
dumb question, why don't we make this a target of the Makefile, and then we can keep `latest-kernel-full-config.sh` as the name of the script?. The parameters to the Makefile target will be the same as with the script.